.github/workflows/update-charts.yml

```diff
@@ -132,3 +127,109 @@ jobs:
         env:
           UPDATECLI_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: "updatecli apply --config ./updatecli/updatecli.d/patch-kubewarden-controller-with-crds-update.yaml --values updatecli/values.yaml"
+
+  major-minor-update:
+    name: Major or minor release updates
+    runs-on: ubuntu-latest
+    needs:
+      - check-update-type
+    if: needs.check-update-type.outputs.update_type == 'major' || needs.check-update-type.outputs.update_type == 'minor'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Check if all components has a release with the same tag
+        uses: actions/github-script@v6
+        with:
+          script: |
+            let repository_split = context.payload.client_payload.repository.split("/")
+            let owner = repository_split[0]
+            const version = context.payload.client_payload.version
+            let repos = ['kubewarden-controller', 'policy-server', 'kwctl']
+
+            for (const repo of repos) {
+              try {
+                await github.rest.repos.getReleaseByTag({owner: owner, repo: repo, tag: version,})
```

Comment:
I wonder if this may fail if the major release that we are comparing got also a patch or an rc bump too for one of the repos.

Reply:
But the patch release will happen after the major release, won't it? The script does not handle concurrent releases. This will require further improvements. The current version will just check if there is the same tag everywhere. It does not care about any other tags.

Reply:
fair enough, true. It will be fine, we will get a PR for the bump to major version, and then a PR for the bump of patch version, that *maybe* gets a git conflict, and can be rebased by hand or fixed by hand, or retriggered etc.